plugins/source/aws/resources/services/ec2/reserved_instances_mock_test.go

```diff
@@ -0,0 +1,32 @@
+package ec2
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2Types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
```

Comment:
nit:
```suggestion
	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
```

Reply:
Fixed here and in a number of other places